x-pack/plugins/enterprise_search/public/applications/enterprise_search_content/components/search_index/pipelines/ml_inference/test_pipeline.tsx

```diff
@@ -151,9 +151,18 @@ export const TestPipeline: React.FC = () => {
                     </strong>
                   </p>
                 </EuiText>
-                <EuiCodeBlock fontSize="m" isCopyable language="json" paddingSize="m">
-                  {`[{"_index":"index","_id":"id","_source":{"${sourceField}":"bar"}}]`}
-                </EuiCodeBlock>
+                {fieldMappings && (
+                  <EuiCodeBlock fontSize="m" isCopyable language="json" paddingSize="m">
+                    {`[{"_index":"index","_id":"id","_source":{${fieldMappings
+                      .map((fieldMapping) => `"${fieldMapping.sourceField}": "REPLACE ME"`)
```

Comment:
I don't really love the `REPLACE ME`, but this felt better than `"bar"`, and a lot better than:
```
"source_a": "bar",
"source_b": "bar",
"source_c": "bar"
```

But I'm open to other ideas.
Also, does this need to hook into internationalization?

Reply:
We should probably hook REPLACE ME into internationalization, yes. It's definitely better than "bar".